deploy/deploy-api.sh

```diff
@@ -22,6 +22,18 @@ fi
 dataset=$1
 project_id=$(jq --raw-output '.project_id' dataset_config/${dataset}/deploy.json)
 
+if [ ! -f "dataset_config/${dataset}/private-key.json" ]; then
+	echo "Private key not found. Send to Terra feature will not work. "
+	echo "See https://github.com/DataBiosphere/data-explorer#one-time-setup-for-export-to-saturn-feature."
```

Comment:
Oops, should be `https://github.com/DataBiosphere/data-explorer#one-time-setup-for-send-to-terra-feature`. Can you fix everywhere in this repo? Thanks